.github/workflows/e2e-tests-workflow.yml

```diff
@@ -99,20 +102,23 @@ jobs:
     name: Run e2e tests without security
     strategy:
       matrix:
-        os: [ubuntu-16.04] # use ubuntu-16.04 as required by cypress: https://github.com/marketplace/actions/cypress-io#important
+        os: [ubuntu-latest]
         java: [14]
     runs-on: ${{ matrix.os }}
     steps:
       - name: Pull and Run Docker
         run: |
-          opensearch_version=$OPENSEARCH_VERSION
-          echo opensearch version: $opensearch_version
-          if docker pull opendistroforelasticsearch/opendistroforelasticsearch:$opensearch_version
+          echo opensearch version: $OPENSEARCH_VERSION
+          if docker pull $OPENSEARCH_DOCKER_IMAGE:latest
           then
-            echo "FROM opendistroforelasticsearch/opendistroforelasticsearch:$opensearch_version" >> Dockerfile
+            echo "FROM $OPENSEARCH_DOCKER_IMAGE:latest" >> Dockerfile
             ## The OpenSearch rest test client uses http by default.
             ## Need to disable the security plugin to call the rest api over http.
-            echo "RUN if [ -d /usr/share/OpenSearch/plugins/opensearch-security ]; then /usr/share/OpenSearch/bin/OpenSearch-plugin remove opensearch-security; fi" >> Dockerfile
+            ## Also need to remove any remaining security settings specified in opensearch.yml
+            echo "RUN if [ -d /usr/share/opensearch/plugins/opensearch-security ]; then /usr/share/opensearch/bin/opensearch-plugin remove opensearch-security; fi" >> Dockerfile
+            echo "RUN sed -n -i '1,/Start OpenSearch Security Demo Configuration/p;/End OpenSearch Security Demo Configuration/,\$p'" >> Dockerfile
+            truncate -s -1 Dockerfile
+            echo -n " /usr/share/opensearch/config/opensearch.yml" >> Dockerfile
```

Comment:
Had to use a workaround to get the full `sed` command to get written to the `Dockerfile` correctly. The purpose of the `sed` command is to remove the extra added security settings in the demo `opensearch.yml` file. Without removing, the cluster has errors, since no plugin is able to recognize the security-related settings (and we are purposefully removing the security plugin in this test case).

Reply:
> no plugin is able to recognize the security-related settings

Is this for testing without security plugin? Or it's just temporary workaround as security plugin not ready?

Reply:
> > no plugin is able to recognize the security-related settings
> 
> Is this for testing without security plugin? Or it's just temporary workaround as security plugin not ready?

This is for testing without security plugin. Sorry if it's confusing - the workaround is related to adding the `sed` cmd to remove the security settings.

Reply:
@ylwu-amzn CI now works with security enabled and disabled: see the latest run [here](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/actions/runs/858565810
)

Reply:
Thanks for clarification.